### PR TITLE
Fix 235 default design

### PR DESCRIPTION
**Pull request by ja-thomas**
See #235

### CODE REVIEW

File: R/mbo.R [hunk 1]
```diff
@@ -17,7 +17,7 @@
 #'   the design must not be transformed before it is passed!
 #'   Functions to generate designs are available in \code{ParamHelpers}: \code{\link[ParamHelpers]{generateDesign}}, \code{\link[ParamHelpers]{generateGridDesign}}, \code{\link[ParamHelpers]{generateRandomDesign}}.
 #'   Default is \code{NULL}, which means \code{\link[ParamHelpers]{generateDesign}} is called and a design
-#'   of size 4 times number of all parameters is created.
+#'   of size 4 times number of all parameters with a maximizing the minimal disctance between design points is created; see \code{\link[lhs]{maximinLHS}}.
```
Comment (jakob-r) [inline R/mbo.R]: We have to overthink this sentence somehow.

Comment (ja-thomas) [inline R/mbo.R]: ups... yes, thats a pretty strange sentence.

...of size 4 times number of all parameters is created. The points are drawn via \code{\link[lhs]{maximinLHS}} to maximize the minimal distance between design points.

**Comment by berndbischl** [comment]
ok fix the docs then merge please

**Comment by jakob-r** [comment]
much nicer now. merge and ignore travis

**Pull request merged**